docs/language/README.md

```diff
@@ -1,6 +1,6 @@
 # `zq` log query language (ZQL)
 
-ZQL is a powerful query language for searching and analyzing event data. It is in many ways optimal for working with [Zeek](https://www.zeek.org/) data, though it can be used to query any data in in [ZNG](../../zng/docs/README.md) or [NDJSON](http://ndjson.org/) format.
+ZQL is a powerful query language for searching and analyzing event data. It is in many ways optimal for working with [Zeek](https://www.zeek.org/) data, though it can be used to query any data in in [ZNG](../formats/README.md) or [NDJSON](http://ndjson.org/) format.
```

Comment:
```suggestion
ZQL is a powerful query language for searching and analyzing event data. It is in many ways optimal for working with [Zeek](https://www.zeek.org/) data, though it can be used to query any data in in [ZNG](../formats/spec.md) or [NDJSON](http://ndjson.org/) format.
```